JASP-Desktop/components/JASP/Widgets/AboutWindow.qml

```diff
@@ -0,0 +1,226 @@
+import QtQuick 2.11
+import QtQuick.Window 2.11
+import QtQuick.Controls 2.5
+import QtQuick.Controls.Styles 1.4
+import QtWebEngine 1.7
+import JASP.Widgets 1.0
+import JASP.Theme 1.0
+
+Window
+{
+	id: aboutWindow
+
+	width:				850
+	height:				500
+
+	maximumHeight: height
+	maximumWidth: width
+	minimumHeight: height
+	minimumWidth: width
+
+	property string labelcolor : "#F99800"
+	property string closebuttoncolor : "#50B0E3"
+
+	visible:			aboutModel.visible
+	onVisibleChanged:	aboutModel.visible = visible
+	title:				"JASP About"
+
+	Image {
+		id:topWave
+		anchors.top:parent.top
+		anchors.left:parent.left
+		anchors.right:parent.right
+		height: parent.height/4
+		source: "qrc:/core/img/jasp-wave-down-blue-120.svg"
+	}
+
+	Rectangle
+	{
+		id: jaspLogo
+
+		width: parent.height/3
+		height: parent.height/5.5
+		anchors.left: parent.left
+		anchors.leftMargin: 50
+		anchors.verticalCenter: parent.verticalCenter
+		Image {
+			anchors.fill: parent
+			fillMode: Image.PreserveAspectFit
+			source: "qrc:/core/img/jasp-logo-black.svg"
+		}
+	}
+
+	Rectangle{
+		id: aboutInfoBox
+
+		anchors.left: jaspLogo.right
+		anchors.leftMargin: 25
+		anchors.right: parent.right
+		anchors.rightMargin: 25
+		anchors.top:topWave.bottom
+		anchors.bottom:bottomWave.top
+		border.color: "gray"
+		border.width:0
+
+		Text {
+			id: copyrightMessage
+			text: aboutModel.jaspCopyrightMessage
+			anchors.left:parent.left
+			anchors.top:parent.top
+			anchors.right: parent.right
+			height:25
+			}
+
+		Label {
+			id:jaspVersionLabel
+
+			text:qsTr("<b>Version: </b>")
+			color: labelcolor
+			anchors.left:parent.left
+			anchors.top:copyrightMessage.bottom
+			height:25
+			width:75
+
+			Text {
+				id: jaspVersionText
+
+				text:aboutModel.jaspVersionString
+				anchors.left:parent.right
+				anchors.top:parent.top
+				height:25
+			}
+		}
+
+		Label {
+			id:buildDateLabel
+
+			text:qsTr("<b>Build on: </b>")
+			color: labelcolor
+			anchors.left:parent.left
+			anchors.top:jaspVersionLabel.bottom
+			height:25
+			width:75
+
+			Text {
+				id: buildDateText
+
+				text:aboutModel.jaspBuildDate
+				anchors.left:parent.right
+				anchors.top:parent.top
+				height:25
+			}
+		}
+
+		Label {
+			id:downloadLabel
+
+			text:qsTr("<b>Download: </b>")
+			color: labelcolor
+			anchors.left:parent.left
+			anchors.top:buildDateLabel.bottom
+			height:25
+			width:75
+
+			Text {
+				id: downloadText
+
+				text:"<a href=\"" + aboutModel.jaspDownloadURL +"\">"+ aboutModel.jaspDownloadURL + "</a>"
+				anchors.left:parent.right
+				anchors.top:parent.top
+				textFormat: Text.StyledText
+				height:25
+				MouseArea {
+					id:mouseAreaDownload
+					anchors.fill: parent
+					onClicked: {
+						Qt.openUrlExternally(aboutModel.jaspDownloadURL)
+					}
+					cursorShape: Qt.PointingHandCursor
+				}
+			}
+		}
+
+		Label {
```

Comment:
It is a problem that you cannot easily copy the citation. In the old About you could select the text in the window and then copy it. We should make it easy for users to cite JASP ;)